src/services/etherspot.js

```diff
@@ -216,6 +217,42 @@ export class EtherspotService {
       });
   }
 
+  async estimateBatch(chain: Chain): Promise<?TransactionFeeInfo> {
```

Comment:
Am i right in thinking that the `estimateBatch` function here is specifically for ENS batch estimation? Could we rename this function to make it ENS specific? The name `estimateBatch` is a bit too ambiguous.